xml/snmp_sysdescr.xml

```diff
@@ -6318,10 +6318,12 @@ Copyright (c) 1995-2005 by Cisco Systems
     <example>Siemens, SIMATIC HMI, KTP600 Basic Mono PN, 6AV6647-0AB11-3AX0, HW:1, FW:V01.06.00</example>
     <example>Siemens, SIMATIC HMI, KTP600 Basic color PN, 6AV6 647-0AD11-3AX0, HW:1, FW:V11.00.02.00</example>
     <param pos="0" name="os.vendor" value="Siemens"/>
```

Comment:
`os.vendor` here and elsewhere should be Microsoft.
`hw.vendor` should, I'm guessing, be `Siemens`?
Can you group the `os.*` and `hw.*` together?